src/style.css

```diff
@@ -1,3 +1,70 @@
 body {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+:root {
+  --first-color: #00acdc;
+}
+
+.header {
+  width: 100%;
+  max-width: 100%;
+  min-width: 1024px;
```

Comment:
It's generally not a good practice to fix the min-width of an element to a pixel value like this. This can lead to horizontal scrolling on smaller screens which is not desirable. Consider using relative units or media queries to handle different screen sizes.